src/main/webapp/consentGroupReview/ConsentGroupReview.js

```diff
@@ -588,6 +592,17 @@ class ConsentGroupReview extends Component {
     });
   };
 
+  requestClarification = () => {
+    this.setState({
+      showRequestClarification: !this.state.showRequestClarification
+    });
+  };
+
+  closeClarificationModal = () => {
+    this.setState({ showRequestClarification: !this.state.showRequestClarification });
+  };
```

Comment:
Here and above - these need to use the updater form because the toggle depends on current state. See the [react docs for an example](https://reactjs.org/docs/react-component.html#setstate). 

>  If the next state depends on the current state, we recommend using the updater function form, instead:
```
this.setState((state) => {
  return {quantity: state.quantity + 1};
});
```